src/app/services.module.ts

```diff
@@ -42,6 +42,7 @@ import { NotificationsService } from 'jslib/services/notifications.service';
 import { PasswordGenerationService } from 'jslib/services/passwordGeneration.service';
 import { PolicyService } from 'jslib/services/policy.service';
 import { SearchService } from 'jslib/services/search.service';
+import { SendService } from 'jslib/services/send.service';
```

Comment:
Okay to include these changes for send with a language commit/PR? Was this intentional?